bpf/bpf_lxc.c

```diff
@@ -756,6 +756,7 @@ __section_tail(CILIUM_MAP_POLICY, LXC_ID) int handle_policy(struct __sk_buff *sk
 	return redirect(ifindex, 0);
 }
 
+#ifdef ENABLE_IPV4
```

Comment:
Let's change the ifdef to `LXC_NAT46` as the caller on `bpf_lxc.c:247` uses this as well.